app/views/curriculums/_language_fields.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class='nested-fields'>
+  <div class="row">
+    <div class="col s4">
+      <%= f.label :Idioma %>
+      <%= f.text_field :language_name %>
+    </div>
+    <div class="col s6">
+      <%= f.label :Nivel_de_conocimiento %>
+      <%= f.select :proficiency, options_for_select([
```

Comment:
Add styles for materialize dropdown.

Reply:
Materialize select is not being used in the whole app and it seems it's really buggy, you can see it has `browser-default` class in every select.